torchaudio/compliance/kaldi.py

```diff
@@ -20,7 +19,7 @@
 ]
 
 # numeric_limits<float>::epsilon() 1.1920928955078125e-07
-EPSILON = torch.tensor(torch.finfo(torch.float).eps, dtype=torch.get_default_dtype())
+EPSILON = torch.tensor(torch.finfo(torch.float).eps)
```

Comment:
It will create EPSILON always on CPU, it is not right?

Reply:
This code needs updating to work on GPU, and we don't officially support GPU for now in torchaudio. I won't do this as part of this PR.